app/src/main/java/com/example/stretchy/database/AppDatabase.kt

```diff
@@ -1,28 +1,270 @@
 package com.example.stretchy.database
 
+import android.annotation.SuppressLint
 import androidx.room.Database
 import androidx.room.RoomDatabase
 import androidx.room.TypeConverters
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
 import com.example.stretchy.database.converter.ActivityTypeConverter
 import com.example.stretchy.database.converter.TrainingTypeConverter
 import com.example.stretchy.database.dao.ActivityDao
 import com.example.stretchy.database.dao.TrainingDao
 import com.example.stretchy.database.dao.TrainingWithActivitiesDao
+import com.example.stretchy.database.data.ActivityType
 import com.example.stretchy.database.entity.ActivityEntity
 import com.example.stretchy.database.entity.TrainingActivityEntity
 import com.example.stretchy.database.entity.TrainingEntity
 
 @Database(
     entities = [TrainingEntity::class, ActivityEntity::class, TrainingActivityEntity::class],
-    version = 1
+    version = 2
 )
 @TypeConverters(TrainingTypeConverter::class, ActivityTypeConverter::class)
 abstract class AppDatabase : RoomDatabase() {
     abstract fun activityDao(): ActivityDao
     abstract fun trainingDao(): TrainingDao
     abstract fun trainingWithActivitiesDao(): TrainingWithActivitiesDao
 
-    companion object{
+    companion object {
         const val NAME = "stretchydb"
+
+        val MIGRATION_1_2 = object : Migration(1, 2) {
+            @SuppressLint("Range")
+            override fun migrate(database: SupportSQLiteDatabase) {
+                fun addActivityOrder() {
+                    database.execSQL("ALTER TABLE training_activities ADD COLUMN activityOrder INTEGER NOT NULL DEFAULT 0")
+
+                    database.execSQL(
+                        "UPDATE training_activities " +
+                                "SET activityOrder = (SELECT COUNT(*) " +
+                                "                    FROM training_activities AS ta " +
+                                "                    WHERE ta.tId = training_activities.tId " +
+                                "                    AND ta.aId <= training_activities.aId)"
+                    )
+
+                    database.execSQL(
+                        "INSERT INTO activity (name, duration, activityType) " +
+                                "VALUES ('', 5, 'BREAK')"
```

Comment:
why a break is added here... fun name does not say anything about it, nor its related